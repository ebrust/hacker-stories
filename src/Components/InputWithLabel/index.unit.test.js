import React from "react";
import renderer from "react-test-renderer";

// import InputWithLabel from "./InputWithLabel";

import InputWithLabel from "./index";

describe("InputWithLabel", () => {
  test("renders snapshot", () => {
    const component = renderer.create(
      <InputWithLabel id="1" value="a">
        Input with Label
      </InputWithLabel>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
