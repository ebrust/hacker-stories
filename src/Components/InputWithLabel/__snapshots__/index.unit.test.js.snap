// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWithLabel renders snapshot 1`] = `
Array [
  <label
    className="label"
    htmlFor="1"
  >
    Input with Label
  </label>,
  "Â ",
  <input
    className="input"
    id="1"
    type="text"
    value="a"
  />,
]
`;
